app/gui/src/project-view/components/ComponentBrowser/filtering.ts

```diff
@@ -271,11 +275,11 @@ export class Filtering {
   }
 
   /** TODO: Add docs */
-  filter(entry: SuggestionEntry): MatchResult | null {
+  filter(entry: SuggestionEntry, additionalSelfTypes?: QualifiedName[]): MatchResult | null {
```

Comment:
Why is it optional? Shouldn't it always include the `Any` type?

Reply:
It can be required, but I don’t think it should always include `Any`. The filtering algorithm knows nothing about standard library and inheritance between types, so hardcoding `Any` at this point would be a mistake.